app/views/grant_submissions/show.html.erb

```diff
@@ -23,6 +23,12 @@
             <dd><%= @grant_submission.grant.name %></dd>
             <dt>Requested Funding</dt>
             <dd><%= number_to_currency @grant_submission.requested_funding_dollars %></dd>
+            <dt>Tags</dt>
+            <dd>
+            <%= SubmissionsTag.joins(:tag).where(grant_submission: @grant_submission).each do |st| %>
+              <%= Tag.find(st.tag_id).name %>
```

Comment:
this isn't working right, it looks like:

Tags
    placement [#<SubmissionsTag id: 2, tag_id: 2, grant_submission_id: 76, created_at: "2018-11-17 20:37:57", updated_at: "2018-11-17 20:37:57">]

Reply:
fixed this, figured out how to do the where clause